lib/FrontendTool/FrontendTool.cpp

```diff
@@ -324,6 +324,8 @@ static void debugFailWithCrash() {
 }
 
 /// Performs the compile requested by the user.
+/// \param Instance Will be reset after performIRGeneration when the verifier
+///                 mode is NoVerify.
```

Comment:
ASan runs take a long time (in fact, there's a rogue timeout somewhere). Let's just go ahead; I saved the link to the build.